.clang-tidy

```diff
@@ -54,8 +54,11 @@ performance-*,
 readability-container-size-empty,
 readability-delete-null-pointer,
 readability-duplicate-include
+readability-make-member-function-const,
```

Comment:
This has a ton of false positives when dealing with objects with pointers inside of them